sample_app/app/models/micropost.rb

```diff
@@ -0,0 +1,18 @@
+class Micropost < ApplicationRecord
+  PARAMS = %i(content image).freeze
+
+  belongs_to :user
```

Comment:
thêm association belongs_to user cho micropost, 1 post thuộc về 1 user. Tương ứng ở model User phải thêm association has_many để mô tả quan hệ one-to-many giữa user và post